### DISCUSSION TITLE
Incorrect Maxmind geolocation

### DISCUSSION THREAD

**Discussion by pdifeo** [rustdesk/rustdesk#7934] [Q&A]
@rustdesk just for your information.

I assure you that I am not in Rome, but 300 km away.

If you use Maxmind to locate the client, the information is wrong. So the restriction to the public server that clients must be in the same city is fallacious.

<img width="708" alt="image" src="https://github.com/rustdesk/rustdesk/assets/8618744/4acac484-2827-4874-b197-287bab11822b">

**Comment by rustdesk** [comment]
Of  course, we knew this, :(. If some one can sponsor us an accurate database, we will be very glad. I will look for better database also, i do not think there is good free one.

**Reply by pdifeo** [reply]
In my opinion, it will be impossible to have an accurate IP DB.

I don't care about the blocking problem because I have a commercial server version. The geolocation of rustdesk is also not good for defining which relay is ‘nearest’.

**Reply by rustdesk** [reply]
Yes

**Reply by rustdesk** [reply]
We will improve this for Pro later, by re-evaluating latency of all devices to all relay servers.

**Reply by rickerk6gdo** [reply]
I am in the same house on different IPs, pretty major issue

**Reply by RockyRen618** [reply]
> We will improve this for Pro later, by re-evaluating latency of all devices to all relay servers.我们稍后将通过重新评估所有设备到所有中继服务器的延迟来改进 Pro 的这一点。

@rustdesk 
hello rustdesk，
Has this problem been resolved? I think it is necessary to consider the packet loss rate as well. Latency is the most important factor, but having low latency and high packet loss rate can also bring a bad experience.

**Reply by bigmoveus** [reply]
any progress regarding this ?

**Comment by jasonketola** [comment]
@pdifeo MaxMind employee here. I'm sorry for the issue you're seeing. The best way to get your information updated in our data is via our corrections form: https://www.maxmind.com/en/geoip-location-correction.

This page provides some additional information on submitting corrections to MaxMind: https://www.maxmind.com/en/geoip-data-correction-request.

**Reply by pdifeo** [reply]
The problem is not only on my IP, which I could easily update its position in your database. How would I update that of my contacts? As you know we also use rustdesk to support customers. So in your opinion I should first tell the customer to update the MaxMind database and then I would be able to connect to his computer? That's not actionable. So that's that.

**Reply by bigmoveus** [reply]
@pdifeo do you get any result ?

**Reply by pdifeo** [reply]
no one

**Comment by bigmoveus** [comment]
is there any ETA for fix this problem ?
